app/src/main/java/org/mozilla/fenix/components/toolbar/interactor/BrowserToolbarInteractor.kt

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.toolbar.interactor
+
+import mozilla.components.ui.tabcounter.TabCounterMenu
+import org.mozilla.fenix.components.toolbar.BrowserToolbarController
+import org.mozilla.fenix.components.toolbar.BrowserToolbarMenuController
+import org.mozilla.fenix.components.toolbar.ToolbarMenu
+
+/**
+ * Interface for the browser toolbar interactor.
+ */
```

Comment:
nit: Lets add more info about the responsibility on the interface

Reply:
I added some extra details, but let me know if you have a suggestion.